topics/neo/index.md

```diff
@@ -0,0 +1,13 @@
+---
+created_by: Hongfei Da, Erik Zhang
+display_name: NEO
+logo: neo.png
+released: July 15, 2016
+short_description: A distributed network for the Smart Economy.
+topic: neo
+related: blockchain, cryptocurrency
+url: https://www.neo.org/
+github_url: https://github.com/neo-project
+documentation_url: http://docs.neo.org/
```

Comment:
This is not a valid field. See: https://github.com/github/explore/blob/master/docs/API.md

Once you remove this line (assuming no other errors), the Travis CI build should pass.